tests/php/ORM/DataObjectSchemaGenerationTest.php

```diff
@@ -67,11 +68,26 @@ function () {
         );
     }
 
+    private function isMySQL8(): bool
+    {
+        $connector = DB::get_conn()->getConnector();
+        if ($connector instanceof MySQLiConnector &&
+            preg_match('#^8\.#', $connector->getVersion())
```

Comment:
This won't work for mariadb, but this is just a temporary measure as per the TODO comments below, so this is fine.